pom.xml

```diff
@@ -1231,6 +1215,13 @@
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
+            <dependency>
```

Comment:
What's this change about? cc @kwart @vbekiaris

Reply:
The Bill Of Materials in the dependencyManagement allows defining one umbrella dependency for all netty components.

Reply:
Thanks, just double-checking we're fine with it. Merging with unrelated test failure.